fiftyone/server/routes/tagging.py

```
@@ -87,3 +128,25 @@ def build_label_tag_aggregations(sample_collection: foc.SampleCollection):
         tags.append(foa.CountValues(path + ".tags"))
 
     return counts, tags
+
+
+async def _get_all_dataset_tags(dataset_name, target_labels):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_get_all_dataset_tags`

(ANN202)